gallery/src/components/NxTextInput/NxTextInputPage.tsx

```diff
@@ -59,9 +59,7 @@ const NxTextInputPage = () =>
             <td>string[]</td>
             <td>No</td>
             <td>
-              Zero or more validation error messages.  If empty or not defined, the field is considered to be valid.
-              If the form is invalid, the entire <code className="nx-code">NxTextInput</code> element will be wrapped in
-              an <code className="nx-code">NxTooltip</code> element, with error styling.
+              Zero or more validation error messages.  If empty or not defined, the field is considered to be valid
```

Comment:
```suggestion
              Zero or more validation error messages.  If empty or not defined, the field is considered to be valid.
              If the field is invalid, it will receive error styling and a tooltip displaying the first error message.
```